libursa/bin/test_ed25519.rs

```diff
@@ -32,16 +32,20 @@ fn main() {
     now = Instant::now();
     for _ in 0..trials {
         unsafe {
-            ffi::crypto_sign_ed25519_detached(signature.as_mut_ptr() as *mut u8,
-                                              0u64 as *mut u64,
-                                              letters.as_ptr() as *const u8,
-                                              letters.len() as u64,
-                                              s.as_ptr() as *const u8);
-
-            ffi::crypto_sign_ed25519_verify_detached(signature.as_ptr() as *const u8,
-                                                     letters.as_ptr() as *const u8,
-                                                     letters.len() as u64,
-                                                     p.as_ptr() as *const u8)
+            ffi::crypto_sign_ed25519_detached(
+                signature.as_mut_ptr() as *mut u8,
+                0u64 as *mut u64,
+                letters.as_ptr() as *const u8,
+                letters.len() as u64,
+                s.as_ptr() as *const u8,
+            );
+
+            ffi::crypto_sign_ed25519_verify_detached(
+                signature.as_ptr() as *const u8,
+                letters.as_ptr() as *const u8,
+                letters.len() as u64,
+                p.as_ptr() as *const u8,
+            )
```

Comment:
+1 I like this formatting better.

Reply:
Yes I've set up the `rustfmt.toml` file for this project so anyone can run **cargo fmt** before a final check-in and PR.

Reply:
See https://github.com/rust-lang/rustfmt/blob/master/Configurations.md